applications/plugins/SofaPython/examples/RegularGridFromMesh.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import Sofa
+import sys
+import os
+import math
+import numpy as np
+
+class CreateGrid(Sofa.PythonScriptController):
```

Comment:
Here I don't understand why you use a PythonScriptController and arg parsing to create a Scene. 

Cannot you do something like:
```python

def createGrid(parentNode, nx, ny, nz, mesh, name): 
        """
        create a grid...blahblah....
        """" 
        m = parentNode.createObject('MeshObjLoader', name="loader", filename=mesh)
        model = parentNode.createObject('OglModel', name="model", src=m.getPathName())
        model.init()
        model.computeBBox()
        
        root.createObject('RegularGrid', name="grid", nx=nx, ny=ny, nz=nz,
                                                            min=model.bbox.minBBox, max=model.bbox.maxBBox)
        root.createObject('MechanicalObject', name="DOFs")
        ## ....        

def createScene(root)!
           try :
                  sys.argv[0]
           except :
                  sys.argv = sys.argv[1].split()
           else:
                  sys.argv = ['-h']
   
           parser = argparse.ArgumentParser(
                description='Loads a 3D model a generates a regular grid using its bounding box')
            parser.add_argument(
                '-x','--gridx', dest='nx', type=int, default=self.nx, help='')
            parser.add_argument(
                '-y','--gridy', dest='ny', type=int, default=self.ny, help='')
            parser.add_argument(
                '-z', '--gridz', dest='nz', type=int, default=self.nz, help='')
            parser.add_argument(
                '-m', '--mesh', dest='mesh', type=str, default=self.mesh, help='')
            parser.add_argument(
                '-o', '--output', dest='name', type=str, default="", help='')
            args = parser.parse_args(argv)
 
            createGrid(root, nx, ny, nz, mesh, name)
```

Reply:
Thanks for the review!
Sure thing, no need for a PythonScriptController here.. I initially copy-pasted the relevant code from a scene of mine that needs a script controller. I'll change it to what you say.